projects/plugins/jetpack/extensions/shared/components/product-management-controls/toolbar-control.js

```diff
@@ -70,11 +79,33 @@ function Product( { onClose, product } ) {
 
 function NewProduct( { onClose } ) {
 	const { clientId, productType } = useProductManagementContext();
+	const siteSlug = useSelect( select => select( membershipProductsStore ).getSiteSlug() );
 	const openBlockSidebar = useOpenBlockSidebar( clientId );
 
+	const isPublishOpen = useSelect(
+		select => select( 'core/edit-post' ).isPublishSidebarOpened(),
+		[]
+	);
+
+	if ( isInCustomizer() ) {
+		return (
+			<MenuItem>
+				{ siteSlug && (
+					<ExternalLink href={ `https://wordpress.com/earn/payments-plans/${ siteSlug }` }>
+						{ getMessageByProductType( 'add a new product', productType ) }
+					</ExternalLink>
+				) }
+			</MenuItem>
+		);
+	}
+
 	const handleClick = event => {
 		event.preventDefault();
 		openBlockSidebar();
+
+		// We need to close the publish sidebar when the user tries to add a new subscription otherwise the block panel is not visible.
```

Comment:
We need to close the publish sidebar when we open the block controls in the POST_EDITOR context. See my other comment.